boards/arm/nucleo_f303k8/nucleo_f303k8.dts

```diff
@@ -51,13 +51,13 @@
 &pll {
 	prediv = <1>;
 	mul = <9>;
-	clocks = <&clk_hsi>;
+	clocks = <&clk_hse>;
```

Comment:
You are still using HSE here instead of HSI.
Using HSI, by setting PLL multiplier to 16 (maximum admissible value), we can reach a SYSCLK of 64 MHz.
That sets APB1 peripheral clock to 32 MHz, and all others to 64 MHz.

Reply:
Not my day. Fixed